diff --git a/node_modules/foxdriver/lib/logger.js b/node_modules/foxdriver/lib/logger.js
index 646d4bd..ce99962 100644
--- a/node_modules/foxdriver/lib/logger.js
+++ b/node_modules/foxdriver/lib/logger.js
@@ -5,7 +5,7 @@ import pkg from '../package.json'
  * levels that are available from `npmlog`
  */
 const NPM_LEVELS = ['silly', 'verbose', 'debug', 'info', 'http', 'warn', 'error', 'chrome', 'firefox']
-npmlog.addLevel('debug', 1000, { fg: 'blue', bg: 'black' }, 'dbug')
+const debug = require('debug')('@benmalka/foxdriver')
 
 export default function Logger (component) {
     const wrappedLogger = {}
@@ -15,8 +15,8 @@ export default function Logger (component) {
      * allow access to the level of the underlying logger
      */
     Object.defineProperty(wrappedLogger, 'level', {
-        get: () => { return npmlog.level },
-        set: (newValue) => { npmlog.level = newValue },
+        get: () => {  },
+        set: (newValue) => {  },
         enumerable: true,
         configurable: true
     })
@@ -26,8 +26,8 @@ export default function Logger (component) {
      */
     for (let level of NPM_LEVELS) {
         wrappedLogger[level] = (...args) => {
-            if (!process.env.DEBUG) return
-            return npmlog[level](prefix, ...args)
+            // @see https://github.com/cypress-io/cypress/issues/7723
+            debug(prefix, ...args)
         }
     }
 
